### PR TITLE
Move guidelines for builder tools and showcase into docs

### PR DESCRIPTION
**Pull request by katomm**
## Checklist
- [x] I have read the [How to Contribute](https://developers.cardano.org/docs/portal-contribute/).
- [x] I have run `yarn build` after adding my changes **without getting any errors**. 

## Updating documentation 
- Move guidelines for builder tools and showcase into the docs/portal-contribute and link from the comments 
- Add a sentence about adding NFT projects like mentioned in #1012

I didn't find a good way to add relative links to the comment section so please note that these link will only work **after** the PR is merged into main:

https://developers.cardano.org/docs/portal-contribute#add-tools-to-builder-tools
https://developers.cardano.org/docs/portal-contribute#add-a-project-to-showcase

### CODE REVIEW

File: docs/portal-contribute.md [hunk 1]
```diff
@@ -46,12 +46,34 @@ If you think something is wrong or something fundamental should change, discussi
 * [Issues in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/issues) 
 * [Discussions in the GitHub Repo](https://github.com/cardano-foundation/developer-portal/discussions)
 
-### Add projects to showcase or builder tools
-Help to add good projects or builder tools. Please note the different guidelines for [adding showcases](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/showcases.js) or [builder tools](https://github.com/cardano-foundation/developer-portal/edit/staging/src/data/builder-tools.js).
+### Add a project to showcase 
+Help to add good projects to the [showcase section](http://developers.cardano.org/showcase/). Please note the guidelines for adding new projects:
+* It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but nothing for this showcase section.
```
Comment (rphair) [inline docs/portal-contribute.md:51]:
```suggestion
* It must be built on Cardano and have a real use case. For example, a forum where people can talk about Cardano is great, but not for this showcase section.
```
Have I understood this correctly? (i.e. a Forum not implemented in Cardano will _not_ be great for the Showcase section and therefore we will have _nothing_ of this kind here)

File: docs/portal-contribute.md [hunk 2]
```diff
@@ -46,12 +46,34 @@ If you think something is wrong or something fundamental should change, discussi
...
+* It has to run on Cardano mainnet.
+* It has to have a running product. (no presale, no protected pages, no coming soon messages)
+* It has to have enough community reputation.
+* It has to provide a unique value from existing showcase items.
```
Comment (rphair) [inline docs/portal-contribute.md:55]:
```suggestion
* It has to provide a unique value distinct from existing showcase items. 
```

File: docs/portal-contribute.md [hunk 3]
```diff
@@ -46,12 +46,34 @@ If you think something is wrong or something fundamental should change, discussi
...
+* It has to run on Cardano mainnet.
+* It has to have a running product. (no presale, no protected pages, no coming soon messages)
+* It has to have enough community reputation.
+* It has to provide a unique value from existing showcase items. 
+* It has to have a stable domain name. (a random Netlify/Vercel domain is not allowed, no URL shortener, no app store links, or similar)
+* The GitHub account that adds the project must not be new. 
+* The GitHub account must have a history/or already be known in the Cardano community.
+* Describe what makes your project special, avoid phrases like "the first this and that". Granular details like which project was first is tribal attribute known to cause rift and conflicts.
+* Please refrain from adding NFT projects unless they are immensely different from what is already listed in terms of utility. We can't list hundrets of NFT. For details please follow or take part in the discussion [here](https://github.com/cardano-foundation/developer-portal/discussions/243). (TLDR: we are adding winners of the CNFT awards)
```
Comment (rphair) [inline docs/portal-contribute.md:60]:
```suggestion
* Please refrain from adding NFT projects unless they are immensely different from what is already listed in terms of utility. We can't list hundreds of NFT projects. For details please follow or take part in the discussion [here](https://github.com/cardano-foundation/developer-portal/discussions/243). (TL;DR: we are adding winners of the [CNFT Awards](https://www.cnft-awards.io/))
```
TL/DR is considered canonical (although TLDR is also acceptable): https://www.merriam-webster.com/dictionary/TL%3BDR

**Review by rphair — COMMENTED** [review]
Really nice to see these stated explicitly in the open 🤓

File: docs/portal-contribute.md [hunk 4]
```diff
@@ -46,12 +46,34 @@ If you think something is wrong or something fundamental should change, discussi
...
+* If you add a project which main component is NFT, please select "nftproject" as tag. (not "nftsupport")
```
Comment (rphair) [inline docs/portal-contribute.md:61]:
```suggestion
* If you add a project with a main component of NFTs, please select "nftproject" as tag. (not "nftsupport")
```

**Review by rphair — COMMENTED** [review]
apologies; Internet went down in the middle of above review 😖

**Comment by katomm** [comment]
Thanks for the suggestions @rphair. 🙏🏻

**Review by rphair — APPROVED** [review]

**Review by os11k — APPROVED** [review]

**Review by fill-the-fill — APPROVED** [review]

**Pull request merged**